resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/BasicTestsSuite.scala

```diff
@@ -103,6 +104,28 @@ private[spark] trait BasicTestsSuite { k8sSuite: KubernetesSuite =>
       .set("spark.files", REMOTE_PAGE_RANK_DATA_FILE)
     runSparkRemoteCheckAndVerifyCompletion(appArgs = Array(REMOTE_PAGE_RANK_FILE_NAME))
   }
+
+  test("Run SparkPi without the default exit on OOM error flag", k8sTestTag) {
```

Comment:
If you are suggesting this PR as a bug fix, you need to add `SPARK-27900` prefix to the newly added test case names.

Reply:
Ok I can do that.